.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
```

Comment:
please include `.vscode/` in your `.gitignore` file. We don't need to keep track of this file in git